lib/parsers/log-parser.js

```diff
@@ -76,10 +85,27 @@ export default class LogParser extends Parser {
 
       match = line.match(ERROR_PATTERN)
       if (match) {
+        let outMessageText
```

Comment:
I think maybe this variable should be named `outputMessage`. The `Text` suffix is a bit redundant.

Reply:
Agreed!